pkg/db/kvstore/kvutils.go

```
@@ -54,3 +57,32 @@ func ToByte(value interface{}) ([]byte, error) {
 		return nil, fmt.Errorf("unexpected-type-%T", t)
 	}
 }
+
+// GetAddress concatenates the Host and Port as single arguement.
+func GetAddress(host string, port int) string {
+	addr := host + ":" + strconv.Itoa(port)
+	return addr
+}
+
+// ValidateAddress validates the host and port values
+func ValidateAddress(address string) error {
+	host, port, err := net.SplitHostPort(address)
+	if err != nil {
+		return errors.New("Invalid Format of address")
```

Don't suppress the original error. Use fmt.Errorf to wrap into new error